packages/experimental/Shadow/src/shadowStyle.ts

```diff
@@ -3,7 +3,7 @@ import { ShadowToken } from '@fluentui-react-native/theme-types';
 import { ColorValue, Platform } from 'react-native';
 
 // For iOS/macOS, the blur property from the token is not the same as the shadow radius value,
-// it needs to be divided by 2 to achieve the same effect.
+// it needs to be divided by 2 to achieve the same effect. See https://github.com/microsoft/apple-ux-guide/blob/gh-pages/Shadows.md
```

Comment:
nit:

Reply:
Sorry didn't see this before it got merged, will fix later